### DISCUSSION TITLE
Issue with Transparency detection

### DISCUSSION THREAD

**Discussion by tkoenig** [ImageMagick/ImageMagick#3429] [Help]
Hi, I am currently looking at upgrading from imagemagick 6.9 to 7, but have an issue regarding transparency detection.

According to the docs(<https://imagemagick.org/script/escape.php>) 
%A should return wether image transparency channel is enabled or not (true/false).

when I run the following command against a JPG file (which is not transparent):
```
convert '39pup.jpg' -auto-orient -quiet -format 'transparent=%A' info:
```
it returns
```
 transparent=Undefined
```

If I run the same exampe against a png file which has transparency, I receive:
```
 transparent=Blend
```

with imagemagick 6.9 I either received `transparent=True` or `transparent=False` as described in the docs.

Is this the expected behaviour, and the docs need to be updated or is there a bug which needs further investigation?

Thanks,
Thomas

**Comment by dlemstra** [comment]
This is correct. You can find more information about porting from version 6 to 7 here: https://imagemagick.org/script/porting.php.

**Reply by tkoenig** [reply]
Thanks for the info! 

So according to these  docs for `-format` only the %Z image property is no longer supported.

Does this mean that we can treat `Undefined` as false for transparency and `Blend` as true? 
Are there other values which could be returned?
